webapp/channels/src/components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -472,311 +468,10 @@ class NotificationsTab extends React.PureComponent<Props, State> {
-                                    />
-                                </label>
-                            </div>
-                            <div className='radio'>
-                                <label>
-                                    <input
-                                        id='pushNotificationMentions'
-                                        type='radio'
-                                        name='pushNotificationLevel'
-                                        checked={pushActivityRadio[1]}
-                                        onChange={this.handlePushRadio.bind(this, NotificationLevels.MENTION)}
-                                    />
-                                    <FormattedMessage
-                                        id='user.settings.push_notification.onlyMentions'
-                                        defaultMessage='For mentions and direct messages'
-                                    />
-                                </label>
-                            </div>
-                            <div className='radio'>
-                                <label>
-                                    <input
-                                        id='pushNotificationNever'
-                                        type='radio'
-                                        name='pushNotificationLevel'
-                                        checked={pushActivityRadio[2]}
-                                        onChange={this.handlePushRadio.bind(this, NotificationLevels.NONE)}
-                                    />
-                                    <FormattedMessage
-                                        id='user.settings.notifications.never'
-                                        defaultMessage='Never'
-                                    />
-                                </label>
-                            </div>
-                            <div className='mt-5'>
-                                <FormattedMessage
-                                    id='user.settings.push_notification.info'
-                                    defaultMessage='Notification alerts are pushed to your mobile device when there is activity in Mattermost.'
-                                />
-                            </div>
-                        </fieldset>
-                    </div>,
-                    pushStatusSettings,
-                    pushThreadsNotificationSelection,
-                );
-
-                submit = this.handleSubmit;
-            } else {
-                inputs.push(
-                    <div
-                        key='oauthEmailInfo'
-                        className='pt-2'
-                    >
-                        <FormattedMessage
-                            id='user.settings.push_notification.disabled_long'
-                            defaultMessage='Push notifications have not been enabled by your System Administrator.'
-                        />
```

Comment:
It seems before we had a case we were checking if push notifications are configured at all in the server? Are we covering that in the new settings?

Reply:
We are still checking the condition, however now i have added a message to say that its disabled

<img width="554" alt="Screenshot 2024-03-20 at 9 07 47 AM" src="https://github.com/mattermost/mattermost/assets/17708702/83f60efc-8981-471b-bcd0-ef7890b3c863">

Reply:
Can we change the text here to include 'Mobile'? Earlier this was under the mobile section so it was clear enough, but now we should make sure people don't get confused about what this is. So the final text can be:

"Mobile push notifications have not been enabled by your System Administrator."

Open to other suggestions.

Reply:
@abhijit-singh  Should the text be in a different formatting so it doesn't get blend in with the rest of the text ?

Reply:
Don't think we need to highlight it too much since the user cannot really take any action on this. I'd be okay with going ahead with what we have for now.

Reply:
<img width="794" alt="Screenshot 2024-03-21 at 4 35 24 PM" src="https://github.com/mattermost/mattermost/assets/17708702/1aea4280-6f7c-436d-a634-22433b0b1a4a">

@abhijit-singh i have added the icon. is this better?

Reply:
@M-ZubairAhmed - Are you open to updating the note text to: "Mobile push notifications haven't been enabled by your system administrator." 

Also, I personally find the ! icon a bit small next to the text.

Reply:
thanks @cwarnermm i updated the test and reverted the icon addition as well